tests/plugins/colorbutton/manual/colorcommand.html

```diff
@@ -0,0 +1,25 @@
+<div id="editor">
+	<p>This is unstyled text.</p>
+
+	<p><span style="color:#c0392b">This text has font color.</span></p>
+
+	<p><span style="background-color:#f39c12">This text has background color.</span></p>
+</div>
+
+<script>
+	var index = 1;
+	CKEDITOR.replace( 'editor', {
+		height: 400,
+		on: {
+			'beforeCommandExec': function( evt ) {
```

Comment:
It would be nice to also verify the color change, so maybe instead of using console logs and notification, you could add some element (outside editor) like small rectangle changing color to the same as text inside the editor?